plugins/importexport/native/NativeImportExportPlugin.inc.php

```diff
@@ -345,13 +356,18 @@ function executeCLI($scriptName, &$args) {
 						$foundErrors = true;
 					}
 				}
-				// If there are any data or validataion errors
+				// If there are any data or validation errors
 				// delete imported objects.
 				if ($foundErrors || !empty($validationErrors)) {
-					// remove all imported issues and sumissions
+					// remove all imported issues and submissions
 					foreach (array_keys($errorTypes) as $assocType) {
 						$deployment->removeImportedObjects($assocType);
 					}
+					echo __('plugins.importexport.common.validationErrors') . "\n";
+					$i = 0;
+					foreach ($validationErrors as $validationError) {
+						echo ++$i . '. L' . $validationError->line . ':' . $validationError->column . ' > ' . $validationError->message . "\n";
+					}
```

Comment:
Besides this, another issue that I had and didn't address here is that when the process fails with an exception, all the errors which were captured previously are gone. It would be good to cover the process with a try/catch and display them, as they might have the answer for the exception.